tests/src/test/scala/cats/tests/ValidatedSuite.scala

```diff
@@ -45,6 +46,9 @@ class ValidatedSuite extends CatsSuite {
            SerializableTests.serializable(ApplicativeError[Validated[String, *], String])
   )
 
+  checkAll("Validated[String, Int]", SelectiveTests[Validated[String, *]].selective[Int, Int, Int])
+  checkAll("Selective[Validated[String, *]]", SerializableTests.serializable(Selective[Validated[String, *]]))
```

Comment:
is this the only pure selective in cats? Are there any other Applicatives that could possibly be raised up?

What about the transformers (Kleisli, StateT, WriterT), some of those can have applicatives when their inner type supports applicative, can we also do selective, I bet...

Reply:
`Validated` is the first one I thought of that can be rigid without being a `Monad`.  `Over` is another (and `Under` would be demoted to `Applicative`, if there is no `Selective`).  Anything else that's `Applicative` without being a `Monad` is a candidate for directly being a `RigidSelective`, but I don't have a list yet.  (I wish we had a means to query for instances!)

The inductive instances on transformers all sound like good candidates, too.